src/scene_battle_rpg2k3.cpp

```diff
@@ -783,14 +805,32 @@ void Scene_Battle_Rpg2k3::ProcessActions() {
 		}
 	}
 
-	if (help_window->IsVisible() && message_timer > 0) {
-		message_timer--;
-		if (message_timer <= 0)
+	if (help_window->IsVisible()) {
+		if ((state == State_SelectActor || state == State_SelectCommand || state == State_AutoBattle) && battle_action_wait == 0) {
 			help_window->SetVisible(false);
+		}
 	}
 
 	switch (state) {
 		case State_Start:
+			if (start_message_shown == 0) {
+				start_message_shown = 1;
+				if (!lcf::Data::terms.battle_start.empty()) {
+					ShowNotification(ToString(lcf::Data::terms.battle_start));
+					SetWait(10, 80);
+				}
+				return;
+			}
+			if (start_message_shown == 1) {
+				start_message_shown = 2;
+				if (!lcf::Data::terms.special_combat.empty() && (Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_initiative || Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_surround || (Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_none && first_strike))) {
+					ShowNotification(ToString(lcf::Data::terms.special_combat));
+					SetWait(30, 70);
```

Comment:
This `Wait(30, 70)` also occurs for back and pincer attack, but with no message displayed.

It does not occur for a normal battle without first strike.

To see this, press and hold escape immediately after the start message box disappears.

Reply:
Fixed.